Assets/VRTK/Prefabs/Resources/Scripts/VRTK_ObjectTooltip.cs

```diff
@@ -55,40 +58,45 @@ public void ResetTooltip()
         /// The UpdateText method allows the tooltip text to be updated at runtime.
         /// </summary>
         /// <param name="newText">A string containing the text to update the tooltip to display.</param>
-        public void UpdateText(string newText)
+        public virtual void UpdateText(string newText)
         {
             displayText = newText;
             ResetTooltip();
         }
 
-        protected virtual void Start()
+        protected virtual void OnEnable()
         {
             ResetTooltip();
+            headset = VRTK_DeviceFinder.HeadsetTransform();
```

Comment:
Since this script now depends on the current SDK Setup's used headset this script needs to be added to the SDK Manager behaviour list to disable & enable on SDK Setup changes:
`AddBehaviourToToggleOnLoadedSetupChange` & `RemoveBehaviourToToggleOnLoadedSetupChange` on `VRTK_SDKManager.instance`. I recommend to add in `Awake` and remove in `OnDestroy`.